tests/unit/bokeh/server/views/test_multi_root_static_handler.py

```diff
@@ -0,0 +1,84 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2023, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations # isort:skip
+
+import pytest ; pytest
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Standard library imports
+from pathlib import Path
+from typing import Any
+
+# External imports
+from tornado.httputil import HTTPConnection, HTTPServerRequest
+
+# Bokeh imports
+from bokeh.application import Application
+from bokeh.util.paths import serverdir
+from tests.support.plugins.managed_server_loop import MSL
+
+# Module under test
+from bokeh.server.views.multi_root_static_handler import MultiRootStaticHandler # isort:skip
```

Comment:
It would be consistent with all other tests to import the entire module, e.g. `as m` or `as bsvm`